cypress/integration/RichTextEditor.spec.ts

```diff
@@ -1228,4 +1224,77 @@ describe('Rich Text Editor', () => {
       });
     }
   });
+
+  describe('Embedded Entry Inlines', () => {
+    const methods: [string, () => void][] = [
+      [
+        'using the toolbar button',
+        () => {
+          cy.findByTestId('toolbar-entity-dropdown-toggle').click();
+          cy.findByTestId('toolbar-toggle-embedded-entry-inline').click();
+          cy.wait(100);
+        },
+      ],
+      [
+        'using the keyboard shortcut',
+        () => {
+          editor().type(`{${mod}}{shift}2`);
+          cy.wait(100);
+        }
+      ]
+    ];
+
+    for (const [triggerMethod, triggerEmbeddedAsset] of methods) {
+      describe(triggerMethod, () => {
+        it('adds and removes embedded entries', () => {
+          editor().click().typeInSlate('hello')
+          triggerEmbeddedAsset();
+          editor().click().typeInSlate('world')
+
+          cy.wait(500);
```

Comment:
Is there any other event/change we can wait for instead of the [magical `cy.wait(500)`](https://docs.cypress.io/api/commands/wait#Time) :) ? it slows our CI runs + It's just a matter of time before this test becomes flaky as well.

If there is no other option then maybe using [cy.waitUntil](https://github.com/NoriSte/cypress-wait-until) is slightly better since it doesn't have to always wait for `500ms`.

Reply:
Probably, but I think this is a larger issue we should address in the rest of our test suite. I'll add a ticket for it.